ui/src/choice_group.tsx

```diff
@@ -62,6 +65,21 @@ export interface ChoiceGroup {
   tooltip?: S
 }
 
+const
+  css = stylesheet({
+    inline: {
+      $nest: {
+        '.ms-ChoiceFieldGroup-flexContainer': {
+          display: 'flex',
+          flexWrap: 'wrap',
+        },
```

Comment:
Prefer to use Fluent way of restyling (using `styles` prop on component) when possible. If Fluent team decides to change the CSS class name, we will not know, but if they change the TS API our build will fail so that we can fix it ourselves.

Reply:
Thanks for the tip. I couldn't add the margin for the subcomponent though as the parent doesn't expose its style props, so I used the `selectors` approach. Let me know if there is a better way of handling this.

I followed this guide https://github.com/microsoft/fluentui/wiki/Component-Styling#-styles--prop